docs/src/HOWTOs/howto-write-type-annotations.md

```diff
@@ -1,7 +1,8 @@
 # How to write type annotations
 
 **Warning:** *This HOWTO discusses how to write type annotations for the new
-type checker [Snowcat][], which is used in Apalache since version 0.15.0.*
+type checker [Snowcat][], which is used in Apalache since version 0.15.0.
+Note that the example specification uses recursive operators, which were deprecated in version 0.23.1.*
```

Comment:
Should we file an issue to ensure that this tutorial eventually gets feature-parity?